requirements/test.txt

```diff
@@ -1,6 +1,6 @@
 case>=1.3.1
 pytest-django>=2.2,<4.0
-pytz>dev
+pytz>=2024.1
```

Comment:
I'm out of my depth here, but this looks safe

Platform is pinned to use a specific version [here](https://github.com/startupgrind/Platform/blob/25afd0be7c05b2cc8b5b1776912e66153f031e1b/requirements.in#L79), `b634448b1e73b9d2402ff515a44bcf95c016b33a`.  That version is [here](https://github.com/bevy/django-celery-beat/commit/b634448b1e73b9d2402ff515a44bcf95c016b33a), which was the last time we made changes in this repo.  So it looks to me like we can merge this without affecting Platform, and can pin and test instances before rolling the change out.